### ISSUE TITLE
Modern PHP typehinting

### ISSUE THREAD

**Issue by palansher** [cvergne/vscode-php-getters-setters#15]
Hello!

Is it possible to generate

```
  /**
     * Get the value of sssss
     */
    public function getSssss(): string
    {
        return $this->sssss;
    }

```

instead of:

```
  /**
     * Get the value of sssss
     *
     * @return  string
     */
    public function getSssss()
    {
        return $this->sssss;
    }
```

according to modern php 7.4/8 specification ?

Thanks for your work!

**Comment by palansher** [comment]
Thank you!
Installed pre-release. Nice one! Surely will use it.

**Comment by alcalyn** [comment]
Here are my getter/setter templates for now:

``` js
module.exports = (property) => `
    public function ${property.getterName()}(): ${property.getType() ? property.getType() : 'mixed'}
    {
        return $this->${property.getName()};
    }
`

module.exports = (property) => `
    public function ${property.setterName()}(${property.getTypeHint() ? property.getTypeHint() + ' ' : '' }\$${property.getName()}): self
    {
        $this->${property.getName()} = \$${property.getName()};

        return $this;
    }
`
```

I added param and return typehints and removed phpdoc.

**Comment by cvergne** [comment]
v1.5.0 pre-release has now been finally released as v1.6.0.
I close this issue, feel free to reopen if necessary.
